files/nrpe/check_object.py

```diff
@@ -277,12 +376,12 @@ def parse_command_line():
   parser.add_option("-p", "--password", dest='password', help='password')
   parser.add_option("-s", "--s3_access_key", dest='s3_access_key', help='username')
   parser.add_option("-k", "--s3_secret_key", dest='s3_secret_key', help='password')
-  parser.add_option("-l", "--s3_host", dest='s3_host', help='host')
+  parser.add_option("-b", "--s3_bucket_url", dest='s3_bucket_url', help='URL to S3 bucket')
+  parser.add_option("-l", "--s3_host", dest='s3_host', help='s3 host')
   parser.add_option("-t", "--tenant", dest='tenant', help='tenant name')
-
   parser.add_option("-d", "--debug", dest='debug', action='store_true', help='Debug mode. Enables logging')
-
-  parser.add_option("-b", "--s3_bucket_url", dest='s3_bucket_url', help='URL to S3 bucket')
+  parser.add_option("-w", "--swift_host", dest='swift_host', help='swift host')
```

Comment:
Since I have a slight OCD about Exact Documentation. This is not exactly the same as the Swift host since it has the `swift/v1` suffix. It's kind of a subsection of a URL. In Ceph docs

https://docs.ceph.com/en/latest/radosgw/keystone/#ocata-and-later

a common term for this is "publicurl" so I think I would most prefer `help=Swift publicurl`.

Reply:
I have updated the code bu taking into account this comment.